woocommerce/woocommerce-android
# Update fluxC version with fix

Do not merge until FluxC changes are merged and changeset is updated accordingly in this PR

Closes: [#2534](https://github.com/wordpress-mobile/WordPress-FluxC-Android/issues/2534)


### Description
Applies these [FluxC changes](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2535) to fix the [visitor stats bug. ](https://github.com/wordpress-mobile/WordPress-FluxC-Android/issues/2534)

### Testing instructions
Check that visitor stats from "Month" tab match visitor stats in the chart for the same months when "Year" is selected. See demo: 


https://user-images.githubusercontent.com/2663464/193774631-53ff786b-daaf-466a-ad6b-545f33b0445e.mp4
